### PR TITLE
python3Packages.aiodiscover: 1.3.3 -> 1.3.4

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 1.3.4

Change log: https://github.com/bdraco/aiodiscover/releases/tag/v1.3.4

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119294` at 1b89ff1c run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built successfully:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.aiodiscover</li>
    <li>python38Packages.pyroute2</li>
    <li>python39Packages.aiodiscover</li>
    <li>python39Packages.pyroute2</li>
  </ul>
</details>
<details>
  <summary>2 suggestions:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `fetchpatch`.
Near pkgs/development/python-modules/aiodiscover/default.nix:5:3:
```
  |
5 | , fetchpatch
  |   ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/python-include-tests.md">python-include-tests</a>

Consider adding a `checkPhase` for tests, or if not feasible, `pythonImportsCheck`.

Near pkgs/development/python-modules/aiodiscover/default.nix:38:0:
```
   |
38 |     description = "Python module to discover hosts via ARP and PTR lookup";
   | ^
```
</li>
  </ul>
</details>

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 119294` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.aiodiscover</li>
    <li>python38Packages.pyroute2</li>
    <li>python39Packages.aiodiscover</li>
    <li>python39Packages.pyroute2</li>
  </ul>
</details>

File: pkgs/development/python-modules/pyroute2/default.nix [hunk 1]
```diff
@@ -1,22 +1,27 @@
-{lib, buildPythonPackage, fetchPypi}:
+{lib
```
Comment (mweinelt) [inline pkgs/development/python-modules/pyroute2/default.nix:1]:
```suggestion
{ lib
```

**Comment by mweinelt** [comment]
Can we get an imports check on aiodiscover?

Also please take a look at the nixpkgs-hammering recommendations.

**Comment by fabaff** [comment]
> Can we get an imports check on aiodiscover?

I guess that we could run some of the tests (those which do not require access to `/etc/resolv.conf`) and use `pythonImportsCheck`.

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 119294` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>5 packages built:</summary>
  <ul>
    <li>home-assistant</li>
    <li>python38Packages.aiodiscover</li>
    <li>python38Packages.pyroute2</li>
    <li>python39Packages.aiodiscover</li>
    <li>python39Packages.pyroute2</li>
  </ul>
</details>

**Review by mweinelt — APPROVED** [review]

**Pull request merged**